homeassistant/components/sense/__init__.py

```diff
@@ -39,34 +44,92 @@
 )
 
 
-async def async_setup(hass, config):
-    """Set up the Sense sensor."""
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Sense component."""
+    hass.data.setdefault(DOMAIN, {})
+    conf = config.get(DOMAIN)
+    if not conf:
+        return True
+
+    hass.async_create_task(
+        hass.config_entries.flow.async_init(
+            DOMAIN,
+            context={"source": SOURCE_IMPORT},
+            data={
+                CONF_EMAIL: conf[CONF_EMAIL],
+                CONF_PASSWORD: conf[CONF_PASSWORD],
+                CONF_TIMEOUT: conf.get(CONF_TIMEOUT),
```

Comment:
Timeout is optional but has a default, so there is always a value. 

I think that it's actually weird that timeout is configurable. We should pick a right timeout as developers.

Reply:
The default timeout is actually to lower as it regularly hits it for me.  I'm going to set it to 10 since that is what actually works and remove it from the config flow

Reply:
On second thought, I'll leave it in for now. I'll likely remove it in a future PR once I've found a value that works well for multiple days.

Reply:
It looks like the timeouts are caused by the update strategy of frequent websocket connect, sip a bit of data, disconnect instead of staying connected.   I'll fix that when I add the sensors.